ucr/ucr-wrapper.sh

```diff
@@ -0,0 +1,30 @@
+# Docker must run with a few special arguments.
```

Comment:
```suggestion
#!/usr/bin/env bash
# Docker must run with a few special arguments.
```

Please add some documentation at the beginning about this script and its use case.

Also, the comment is already good but please explain why these comments must be set. What is their meaning?